bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -222,8 +221,11 @@ fn enable_logging_subsystem(config: &Arc<Config>) {
 
     let subscriber = tracing_subscriber::fmt::Subscriber::builder()
         .with_env_filter(
-            EnvFilter::try_new(config.log_level)
-                .expect("could not parse log level in configuration"),
+            EnvFilter::try_new(format!(
+                "{:?},hyper=warn,reqwest=warn,rustls=warn,datadog-trace-mini-agent=warn",
+                config.log_level
+            ))
+            .expect("could not parse log level in configuration"),
```

Comment:
If customers sets log level to `debug`, some crates might spam tons of logs, this allows us to set specific logging level per crate

Reply:
Solved in #329